apis/extension/node.go

```diff
@@ -125,9 +140,45 @@ func GetNodeCPUSharePools(nodeTopoAnnotations map[string]string) ([]CPUSharedPoo
 	if err != nil {
 		return nil, err
 	}
+
+	cpusReserved := ""
+	if val, ok := nodeTopoAnnotations[CPUsReservedByNode]; ok {
+		cpusReserved = val
+	}
+
+	cpuSharePools = filterOutCpusByNodeAnno(cpuSharePools, cpusReserved)
+
 	return cpuSharePools, nil
 }
 
+// filterOutCpusByNodeAnno filter out cpus that reserved by annotation of node.
+func filterOutCpusByNodeAnno(cpuSharePools []CPUSharedPool, cpusReservedByNodeAnno string) []CPUSharedPool {
+	if cpusReservedByNodeAnno == "" {
+		return cpuSharePools
+	}
+
+	newCPUSharePools := make([]CPUSharedPool, len(cpuSharePools))
+	for idx, val := range cpuSharePools {
+		newCPUSharePools[idx] = val
+	}
+
+	reservedCPUs, err := cpuset.Parse(cpusReservedByNodeAnno)
```

Comment:
If the users reserved cpus with `KoordReserved .ReservedCPUs`, it's fine.

But if the users reserved cpus  with `KoordReserved.ReservedResources` and the reseved CPUs quantity such as `6000m`, so do we need to calc which cpus should be reserved like kubelet's CPU Manager policy `static`?

And If users both set `KoordReserved .ReservedCPUs` and  `KoordReserved.ReservedResources`, how it works?

Reply:
`KoordReserved.ReservedCPUs` have higher priority if both those two all specficed, total cpu quantity will be caculated first, and then the cpu's available to the be pod will be calculated.